pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.black]
```

Comment:
Can you move this to `setup.cfg` ? That's the place all the pre-commit configs are present. I think black config should go there as well.

Reply:
I don't think `poetry` recognizes `setup.cfg` file

Reply:
It does not, no.